Cargo.toml

```diff
@@ -24,11 +24,12 @@ ruby-static = ["rb-sys/ruby-static"]
 
 [dependencies]
 magnus-macros = { version = "0.1.0", path = "magnus-macros" }
-rb-sys = { version = "~0.9.34", default-features = false, features = ["bindgen-rbimpls", "bindgen-deprecated-types"] }
+rb-sys = { version = "~0.9.37", default-features = false, features = ["bindgen-rbimpls", "bindgen-deprecated-types"] }
 
 [dev-dependencies]
 magnus = { path = ".", features = ["embed", "rb-sys-interop"] }
 
 [package.metadata.docs.rs]
-all-features = true
+no-default-features = true
```

Comment:
While this isn't a required diff, I think making the features explicit here makes it clear that we need the `embed` feature for doctests to run correctly.

Reply:
The `all-features = true` here is so all features are enabled when [docs.rs](docs.rs) builds the docs, so that all features are displayed in the documentation. This doesn't impact the doctests. It's lines 29-30 that enable the `embed` feature for tests.

Reply:
Makes sense. Reverted!